Makefile

```diff
@@ -7,11 +7,14 @@ SUBDIR = sys lib bin usr.bin etc include contrib
 
 all: install
 
-format: setup
 build: setup
 install: build
 distclean: clean
 
+format:
+	echo "'make format' is deprecated. Use 'verify-format.sh --fix' instead."
```

Comment:
Why not just call the command here?

Reply:
Okay, now we're calling `verify-format.sh` here.